reference/content-types/smart_content.rst

```diff
@@ -46,110 +52,282 @@ Parameters
         choose between the elements in this collection. The selected value is
         also passed to the HTML template.
 
+Return Value
+------------
+
+This values are available in the *view* variable in the twig templates.
+
+.. list-table::
+    :header-rows: 1
+
+    * - Name
+      - Type
+      - Description
+    * - dataSource
+      - string
+      - Uuid of data-source
+    * - includeSubFolders
+      - bool
+      - Is TRUE if subfolders will be crawled
+    * - category
+      - string[]
+      - Selected categories
+    * - tags
+      - string[]
+      - Selected tags.
+    * - sortBy
+      - string
+      - Selected sort column
+    * - sortMethod
+      - string
+      - Selected sort method - ASC or DESC
+    * - presentAs
+      - string
+      - selected present as value
+    * - limitResult
+      - string
+      - Selected limit for result
+    * - page
+      - int
+      - Current page number
+    * - hasNextPage
+      - bool
+      - Is TRUE if another page exists
+
+The "content" values depends on the DataProvider and will be described in the next section.
+
+DataProvider
+------------
+
+These providers are predefined for Sulu-Entities.
+
+Content Pages
+~~~~~~~~~~~~~
+
+Alias: "content"
+
+This provider filters content pages. You can choose as datasource a parent page
+in which folder and (configurable) recursive subfolders will be searched.
```

Comment:
actually we don't talk about folders, but pages. So I would write `You can choose a parent page as data source, whose child pages will be filtered by the DataProvider`.